tensorflow/workspace.bzl

```diff
@@ -508,6 +508,14 @@ def tf_workspace(path_prefix="", tf_repo_name=""):
   )
 
   native.new_http_archive(
+    name = "lmdb",
```

Comment:
I was looking at Bazel manual and `new_http_archive` doesn't seem to have a field called `licenses`. Shall I switch to use `web_library_external` which has the license field? I've added a license in `third_party/lmdb.BUILD` though